python/observation-publisher/publisher/prediction_log_parser.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+from google.protobuf.internal.well_known_types import ListValue, Struct
+from merlin.observability.inference import InferenceSchema, ValueType
+
+PREDICTION_LOG_TIMESTAMP_COLUMN = "request_timestamp"
+
+
+@dataclass
+class PredictionLogFeatureTable:
+    columns: List[str]
+    rows: List[List[Union[np.int64, np.float64, np.bool_, np.str_]]]
+
+
+@dataclass
+class PredictionLogResultsTable:
+    columns: List[str]
+    rows: List[List[List[Union[np.int64, np.float64, np.bool_, np.str_]]]]
```

Comment:
the type should be `List[List[Union[np.int64, np.float64, np.bool_, np.str_]]]` right ?